File: Week -2/Qs4.py
n = int(input("Enter number of elements in the array: "))
arr = []
print(f"Enter {n} elements: ")
for i in range(n):
    element = int(input())
    arr.append(element)
pos = int(input(f"Enter the position of the element to delete (1 to {n}): "))
if pos < 1 or pos > n:
    print(f"Invalid position! Please enter a position between 1 and {n}")
else:
    arr.pop(pos - 1)
    print("Array after deletion: ")
    for element in arr:
        print(element, end=" ")
    print()

File: Week -2/Qs4.c
#include <stdio.h>

int main() {
    int arr[100], n, i, pos;

    printf("Enter number of elements in the array: ");
    scanf("%d", &n);

    printf("Enter %d elements: ", n);
    for(i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    printf("Enter the position of the element to delete: ");
    scanf("%d", &pos);
    for(i = pos - 1; i < n - 1; i++) {
        arr[i] = arr[i + 1];
    }
    n--;
    printf("Array after deletion: ");
    for(i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }

    return 0;
}
